src/main/resources/data/totemhunt/lang/en_us.json

```diff
@@ -1,6 +1,7 @@
 {
 	"game.totemhunt.coral_mall_totem_hunt": "Coral Mall Totem Hunt",
 	"game.totemhunt.lost_cavern_totem_hunt": "Lost Cavern Totem Hunt",
+	"game.totemhunt.the_taterooms_totem_hunt": "The Taterooms Totem Hunt",
```

Comment:
Could this map be named `Taterooms` instead (with the identifier updated as well) for consistency with other map names?